src/icons/ArrowIcon.js

```diff
@@ -4,7 +4,7 @@ import { Icon } from '../components/Icon'
 
 const ArrowIcon = (props) => (
   <Icon { ...props }>
-    <g transform='translate(0, -1)'>
+    <g transform='translate(0.75, -1)'>
```

Comment:
oh ok - they still have to be centered in the viewbox right? I wanted to avoid things like `translate(-6.470246, -12.598699)` - also it is nice to have it on the parent component so we can apply this rotation to any icon - I just dont know how to avoid the manual adjustment of the svg viewbox or translations to center the images..